applications/services/storage/storage.h

```diff
@@ -15,12 +15,16 @@ extern "C" {
 #define STORAGE_INT_PATH_PREFIX "/int"
 #define STORAGE_EXT_PATH_PREFIX "/ext"
 #define STORAGE_ANY_PATH_PREFIX "/any"
-#define STORAGE_APP_DATA_PATH_PREFIX "/data"
-#define STORAGE_APP_ASSETS_PATH_PREFIX "/assets"
 
 #define INT_PATH(path) STORAGE_INT_PATH_PREFIX "/" path
 #define EXT_PATH(path) STORAGE_EXT_PATH_PREFIX "/" path
 #define ANY_PATH(path) STORAGE_ANY_PATH_PREFIX "/" path
+
+#define STORAGE_APPS_DATA_STEM EXT_PATH("apps_data")
+#define STORAGE_APPS_ASSETS_STEM EXT_PATH("apps_assets")
```

Comment:
moved from storage_i.h "APPS_DATA_PATH" and "APPS_ASSETS_PATH" to here to avoid duplicated definition with "apps_data" explicitly in string. not sure if its fine to be exposed to public api, if it gives wrong idea